wpimath/src/main/native/include/frc/estimator/MecanumDrivePoseEstimator.h

```diff
@@ -83,7 +83,7 @@ class WPILIB_DLLEXPORT MecanumDrivePoseEstimator {
    */
   MecanumDrivePoseEstimator(
       const Rotation2d& gyroAngle, const Pose2d& initialPose,
-      const MecanumDriveWheelPositions& wheelPositions,
+      MecanumDriveWheelPositions wheelPositions,
```

Comment:
Aggregates larger than a double should be passed by const ref.

Reply:
Should this also apply to `wpi::array<SwerveModulePosition, NumModules>`?

Reply:
Yes.

Reply:
Done.